run-build.sh

```diff
@@ -13,28 +13,25 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
   [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+REPOROOT="$DIR"
 OLDPATH="$PATH"
 
-REPOROOT="$DIR/../.."
+ARCHITECTURE="x64"
```

Comment:
This value is able to be passed in in the `.ps1` script, why not here?

I get that we don't have x86 unix builds, but it is worth calling out the difference between the scripts somehow.